app/models/invite_email_ect.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
 
 class InviteEmailEct < TrackedEmail
+  INVITES_SENT_FROM = Date.new(2021, 8, 20).beginning_of_day
```

Comment:
We've added this date because that's when we're planning to start sending emails from?

Reply:
More or less, the exact date is still in the works, I think, but yeah.